dlpy/model.py

```diff
@@ -699,6 +699,10 @@ def train(self, table, attributes=None, inputs=None, nominals=None, texts=None,
                               force_equal_padding=force_equal_padding, data_specs=data_specs, n_threads=n_threads,
                               target_order=target_order)
 
+        # the recordSeed option must not be specified in order to disable it, contrary to the Viya documentation
+        if (record_seed == 0) or (record_seed == None):
```

Comment:
please remove the ()s
can you also use "del parameters['record_seed'] 
thanks

Reply:
Aren't we using () for compound logical statements?  I did a quick scan of the code and found others.  Here's and example:

if (int(im_summary.maxWidth) != output_width) or \
                (int(im_summary.maxHeight) != output_height):

Without the () I find the code harder to read.

Reply:
I agree that reading is difficult.
python style does not use ().
A lot slips through my attention, feel free to slip yours as well

Reply:
Requested changes have been made.